apps/prairielearn/src/pages/administratorInstitutions/administratorInstitutions.html.ts

```diff
@@ -89,3 +101,112 @@ export function AdministratorInstitutions({
+            </button>
+          </div>
+          <div class="modal-body">
+            <form name="add-institution" id="add-institution" method="POST">
+              <input type="hidden" name="__action" value="add_institution" />
+              <input type="hidden" name="__csrf_token" value="${csrf_token}" />
+              <div class="form-group">
+                <label for="short_name">Short name</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="short_name"
+                  name="short_name"
+                  placeholder="Short name"
+                />
+                <small id="short_name_help" class="form-text text-muted">
+                  Use an abbreviation or short name. E.g., "UIUC" or "Berkeley".
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="long_name">Long name</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="long_name"
+                  name="long_name"
+                  placeholder="Long name"
+                />
+                <small id="long_name_help" class="form-text text-muted">
+                  Use the full name of the university. E.g., "University of Illinois
+                  Urbana-Champaign".
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="display_timezone">Timezone</label>
+                <select class="form-control" id="display_timezone" name="display_timezone">
+                  <option value="" selected disabled hidden>Timezone</option>
+                  ${Array.from(availableTimezones.keys()).map(
+                    (timezoneName) => html`
+                      <option value="${timezoneName}">${timezoneName}</option>
+                    `,
+                  )}
+                </select>
+                <small id="display_timezone_help" class="form-text text-muted">
+                  The allowable timezones are from the
+                  <a
+                    href="https://en.wikipedia.org/wiki/List_of_tz_database_time_zones"
+                    target="_blank"
+                    >tz database</a
+                  >. It's best to use a city-based timezone that has the same times as you. E.g.,
+                  "America/Chicago".
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="uid_regexp">UID regexp</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="uid_regexp"
+                  name="uid_regexp"
+                  placeholder="UID regexp"
+                />
+                <small id="uid_regexp_help" class="form-text text-muted">
+                  Should match the non-username part of students' UIDs. E.g., @example.com$.
```

Comment:
Nit (this wasn't present in the screenshot I sent earlier, but it's now how the form looks on PT). The backslash is important so that the dot matches a literal `.` and not just "any character".

```suggestion
                  Should match the non-username part of students' UIDs. E.g., @example\\.com$.
```

Reply:
Updated locally in commit [ed4f9f4](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/ed4f9f44057278fce12a5e2de1948446144ae6c6).